packages/odyssey-react/src/components/Tag/Tag.module.scss

```diff
@@ -12,20 +12,20 @@
 
 .list {
   display: inline-block;
-  margin-block-end: -$spacing-xs;
-  margin-inline-start: $spacing-xs;
+  margin-block-end: calc(-1 * var(--ListSpaceMarginBlockEnd));
```

Comment:
nice, yea because we're using a css custom property now (as opposed to compile time only sass variables) we have to rely on CSS `calc` at run time good change